src/app/pages/office/cat-d/office.cat-d.module.ts

```diff
@@ -35,6 +36,7 @@ import { OfficeCatDPageRoutingModule } from './office.cat-d-routing.module';
     PassFinalisationComponentsModule,
     TestFinalisationComponentsModule,
     WaitingRoomToCarComponentsModule,
+    OfficeCatBPageModule,
```

Comment:
rathern then adding your new component into officecatb module and injecting that everywhere, can you add it to the office.components module